apps/backend/src/index.ts

```diff
@@ -24,6 +25,10 @@ new Elysia()
       },
     }),
   )
+  .onError(({ error, code }) => {
```

Comment:
**question:** Does it work when error occurs in APIs inside? Not sure, if it is not dependent on order of chaining.

Reply:
I dont know how to solve it.

This handler will capture all exceptions from user-defined codebase.

Reply:
No problem, we can always solve it later. I just wasn't sure about this one.